### DISCUSSION TITLE
Adding type of bill as a valid billing_code_type

### DISCUSSION THREAD

**Discussion by iGokul-G** [CMSgov/price-transparency-guide#404] [Ideas]
Payers may have  payment arrangements for institutional providers with once of attribute as Type of bill.  ex.   all outpatient services  ( Type of bill equal to 131-149 , 710 - 739 , 831-839 , 851-859 etc. ) 

https://med.noridianmedicare.com/web/jea/topics/claim-submission/bill-types

**Comment by narenlulla** [comment]
The first two characters are important, the third character shows if the bill is first/final/interim/corrected etc. 11 equals Facility In patient, 13 equals Facility out-patient. We have many contracts with different % off charges for InPatient v/s OutPatient bills
